scripts/auth_collections.sh

```diff
@@ -17,10 +17,10 @@
 # authQuery
 #############
 # Purpose:
-#     Provide a very concise way to fetch collections of a2 auth resources at the API level.
+#     Provide a very concise way to fetch collections of Automate auth resources at the API level.
 #
 # Usage:
-#     a2 [ <options> ] <collection> [ <name> <value> ]
+#     authQuery [ <options> ] <collection> [ <name> <value> ]
```

Comment:
This is the key change for this PR -- resyncing these doc-comments with the function name change (which happened a while ago).